test/unit/DataLoader.test.js

```diff
@@ -0,0 +1,165 @@
+/* eslint-env mocha */
+import React from 'react';
+import sinon from 'sinon';
+import axios from 'axios';
+import { expect } from 'chai';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+import DataLoader from './../../src/app/components/DataLoader/DataLoader';
+import Actions from './../../src/app/actions/Actions';
+import appConfig from '@appConfig';
+
+Enzyme.configure({ adapter: new Adapter() });
```

Comment:
You don't need to add this in every test file. If you define this in `/test/helpers/browser.js`, then it will be configured globally.